src/Sylius/Bundle/CoreBundle/Fixture/Factory/PaymentMethodExampleFactory.php

```diff
@@ -114,7 +118,9 @@ protected function configureOptions(OptionsResolver $resolver)
             ->setDefault('description', function (Options $options) {
                 return $this->faker->sentence();
             })
-            ->setDefault('gateway', 'offline')
+            ->setDefault('gatewayName', 'Offline')
+            ->setDefault('gatewayFactory', 'offline')
+            ->setDefault('gatewayConfig', [])
```

Comment:
Yes, but I don't know is it needed to introduce another example factory for such a simple case.